app/controllers/users/delete_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Users
-  class DeleteController < ReauthnRequiredController
+  class DeleteController < ApplicationController
```

Comment:
I noticed reviewing this that this controller doesn't track any analytics events. How'd you feel about adding those here?

Reply:
Happy to add them! Good idea, thanks